flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -216,11 +218,16 @@
     private final DeploymentStateTimeMetrics deploymentTimeMetrics;
 
     private final BoundedFIFOQueue<RootExceptionHistoryEntry> exceptionHistory;
+    private JobGraphJobInformation jobInformation;
+    private ResourceCounter desiredResources = ResourceCounter.empty();
 
     private final JobManagerJobMetricGroup jobManagerJobMetricGroup;
 
+    private final Duration slotIdleTimeout;
+
```

Comment:
It's actually a check interval, not the timeout, right?
Should we then call it `slotIdlenessCheckInterval`?

Reply:
It's the slot timeout, but it's being used as an interval.
If you'd name it "interval" then you'd have to wonder what this is interval is based on.
I'd keep it as-is.

Reply:
> If you'd name it "interval" then you'd have to wonder what this is interval is based on.

I usually expect variables to be named after their role, not source of their value (this I can lookup).
(but I marked renaming as nit)